docs/tutorials/get-started-with-ngo.md

```diff
@@ -188,19 +194,48 @@ This section guides you through testing the RPCs you added in the earlier sectio
 2. Stop the player.
 3. Launch the client and server together in a terminal, as shown in [Testing the command line helper](command-line-helper).
 
+:::tip
+
+Instead of using the command line helper script, you can use Multiplayer Play Mode package, which lets you run multiple instances of the Unity Editor to test multiplayer functionality. See [MPPM](../../tools/mppm.md) to learn more.
```

Comment:
It might help to call out here that MPPM is 23.1 editor version support onwards. I realise that info will be in the link but presenting that info here as well would be a better user experience.

Reply:
Good point. I'll add a note 🙂